libraries/suggestion-input-view/src/main/java/com/trendyol/suggestioninputview/SuggestionInputEditText.kt

```diff
@@ -0,0 +1,31 @@
+package com.trendyol.suggestioninputview
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Rect
+import android.util.AttributeSet
+import androidx.appcompat.widget.AppCompatEditText
+
+class SuggestionInputEditText(context: Context, attrs: AttributeSet) : AppCompatEditText(context, attrs) {
```

Comment:
You should implement all default constructors seperately and make this class internal to prevent reach from outside of the library.

Reply:
> You should implement all default constructors seperately and make this class internal to prevent reach from outside of the library.

done